pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -181,7 +191,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return fmt.Errorf("log-storage-controller failed to watch the ConfigMap resource: %w", err)
 	}
 
-	if err := utils.AddServiceWatch(c, render.ElasticsearchServiceName, render.ElasticsearchNamespace); err != nil {
+	if err := utils.AddServiceWatch(c, render.ElasticsearchInternalServiceName, render.ElasticsearchNamespace); err != nil {
```

Comment:
I think this should now be both since they are created in this controller now. I've added watches for ElasticsearchServiceName and  esgateway.ServiceName now that we've gone away from the `internal` naming.